src/converter.ts

```diff
@@ -23,20 +22,16 @@ export interface ConverterOption {
 }
 
 export interface ConvertResult {
-  output: string
-  path: string
-  rendered: TemplateResult['rendered']
-  result: Buffer | TemplateResult['result']
+  file: File
+  newFile: File
+  template: TemplateResult
 }
 
 export class Converter {
   readonly options: ConverterOption
 
   constructor(opts: ConverterOption) {
     this.options = opts
-
-    if (opts.type === ConvertType.pdf && opts.output === '-')
-      error('PDF cannot output to stdout.')
```

Comment:
By this change, we no longer restrict to output PDF into stdout. The default output path will decide by where came input from.